specs/pvs/corla.pvs

```diff
@@ -2800,23 +2803,112 @@ BEGIN
     UPDATE_BALLOT_MARKS,
     UNDEFINED
   }
-%%% \todokiniry{Specifications for the transport types.}
+%%% We specify the content of data sent to/from the server either with
+%%% a specification of a URL parameter-based communication or using a
+%%% JSON object. We do not wish to precisely specify the JSON, as
+%%% encoding is non-unique due to the existenxce of semantically
+%%% equivalent encodings (as with any serialization technique) and
+%%% across libraries. Instead, we specify data-in-transit implicitly
+%%% via the function signatures in the theories above that focus on
+%%% the endpoint callback (the various \texttt{dashboard} theories)
+%%% and we name the semantially relevant fields using a list of
+%%% strings using the \texttt{json\_spec} type.
   json_spec: TYPE = list[string]
+%%% The \texttt{nothing} constant specifies that no content is
+%%% transmitted for a given communication with the server. This
+%%% commonly happens, e.g., when performing a \texttt{GET} on an
+%%% endpoint.
   nothing: json_spec = null
-%%% This is the standard response that the server gives to all public
-%%% inbound events to permit the UI (the view of the MVC architecture)
-%%% to know what state the server is in, which UI elements should be
-%%% enabled (and thus what transitions are legal).
+%%% The \texttt{server\_response} concept is the standard/normal
+%%% response that the server gives to all public inbound events to
+%%% permit the UI (the view of the MVC architecture) to know what
+%%% state the server is in, which UI elements should be enabled (and
+%%% thus what transitions are legal).  We call this kind of behavior
+%%% in the server ``normal behavior''. In all normal behavior cases
+%%% the server will answer with an HTTP response code of
+%%% \texttt{OK\_200}.
   server_response: json_spec =
     (: "server_state", "ui_events" :)
-% Note that this type is used as a response for both ballot manifest
-% and CVR uploads.
+%%% The \texttt{upload\_response} concept is used to encode the
+%%% variety of response status the server can give to an upload. Note
+%%% that the \texttt{upload\_response} type is used as a response for
+%%% both ballot manifest and CVR uploads. These responses can happen
+%%% synchronously (our plan for \emph{stage-1} deliverable) or
+%%% asynchronously (our plan for a later deliverable). Asynchronous
+%%% updates happen via a call to the \texttt{refresh...} endpoints.
   upload_response: json_spec =
-    % legal values for status are:
-    % "too_late", "data_transmission_interrupted", "upload_sucessful",
-    % "checking_hash", "hash_wrong", "hash_verified", "parsing_data",
```

Comment:
perhaps s/hash_wrong/hash_mismatch/ here and elsewhere, since we don't know if they got or entered the wrong value, or if there was file corruption along the way.